src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static int salary;
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String DATA_INDEX = " ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromLocal = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToLocal = LocalDate.parse(dateTo, formatter);
+        StringBuilder reportSalary = new StringBuilder("Report for period " + dateFrom + " - "
+                + dateTo);
+        for (String name : names) {
```

Comment:
you instantiated string builder and use string concatenation. 
may be avoided like: 
```
StringBuilder sb = new StringBuilder();
sb.append("Report for period ")
.append(dateFrom)
.append(" - ")
.append(dateTo);
```